src/c++/perf_analyzer/docs/cli.md

```diff
@@ -540,8 +540,8 @@ Analyzer.
 
 Specifies the file where trace output will be saved.
 
-If `--trace-log-frequency` is also specified, this argument value will be the
-prefix of the files to save the trace output. See `--trace-log-frequency` for
+If `--log-frequency` is also specified, this argument value will be the
```

Comment:
Is this a correction or did we change the option here? Usually this type of change is reserved for a major version change because it can break user workflows or automation.

Reply:
The option is named `log-frequency` in the command line parser, and that wasn't changed in this PR

Reply:
interesting. well then this is fine for me.